web-ui/src/main/java/org/archcnl/ui/input/MappingListEntryLayout.java

```diff
@@ -24,8 +24,8 @@ public MappingListEntryLayout(MappingListEntry entry, InputContract.Remote input
         text.setWidth("100%");
         addAndExpand(text);
         if (entry.isLeaf() && entry.isAlterable()) {
-            editButton = new Button(EDIT_TEXT, click -> editButtonPressed());
-            deleteButton = new Button(DELETE_TEXT, click -> deleteButtonPressed());
+            editButton = new Button(new Icon(VaadinIcon.PLUS), click -> editButtonPressed());
```

Comment:
The + icon is not a good representation of what the button does. Instead, we should use a pen icon.

Reply:
Icon was changed to Pen Icon